### ISSUE TITLE
Android: "Flare" directory should be created if it does not exist

### ISSUE THREAD

**Issue by dorkster** [flareteam/flare-engine#1870]
With the way modern Android works, users likely don't even have permission to drop the game files into the app's data directory.

The game already looks for a "Flare" directory on internal and external storage. We just need to ensure it's created.

**Comment by BRBsoup** [comment]
So is it possible to install on Android 13? Which directory should be used?

**Comment by dorkster** [comment]
> So the directions for Android install should really say to add game data to "internal storage/Flare/mods" or "/storage/emulated/0/Flare/mods"

Yes. The game looks for the "Flare" directory first, so the game will suggest it for installing data. Provided it exists.

**Comment by BRBsoup** [comment]
> > So the directions for Android install should really say to add game data to "internal storage/Flare/mods" or "/storage/emulated/0/Flare/mods"
> 
> Yes. The game looks for the "Flare" directory first, so the game will suggest it for installing data. Provided it exists.
It would be great to just clarify that at least here:

https://github.com/flareteam/flare-engine/wiki/Android-Port

I was wondering why the install directions said that 2 paths would be suggested and I only saw one. Since as of now those directions need to be read and followed anyway, its as important as the suggested directory path which the game shows. Given that with at least Android 13 the data folder can't easily be written to, after much trying to get the game installed, it was only after finding this issue ticket, plus the "logging" sections of the directions, that I went ahead and made the directory and was able to get it working! Thanks so much for making such a good quality game available cross-platform and especially for Android! I realize that it can be a pain for Android app porting/development, especially with Google making changes and/or increasingly locking down the system in such annoying ways...
